CougSat1-PMIC/src/PMICConfiguration.h

```diff
@@ -149,4 +149,48 @@
 #define THERM_CAL_PV3_FRONT_BOTTOM_270K (0.158)
 #define THERM_CAL_PV3_FRONT_BOTTOM_350K (0.852)
 
+
+// Comparison Constant for Thermistor readings in celcius
+#define TEMP_MIN (0)
+#define TEMP_MAX (50)
```

Comment:
Min and max of what? Battery? Also naming conventions:
THRES_BATT_TEMP_LOW (0)
THRES_BATT_TEMP_HIGH (50)
THRES_REG_TEMP_HIGH (85)